d8/d8forms/d8forms.module

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/**
+ * @file
+ * Contains d8forms.module..
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+define('D8FORMS_MODULE_NAME', 'd8forms');
```

Comment:
Hmm, this is the first time I'm seeing the module's machine name encoded this way. I don't really understand the reasoning behind this.